examples/app/crossfilter/models/styleable_box.coffee

```diff
@@ -39,15 +39,10 @@ class StyleableBox extends BaseBox.Model
   type: "StyleableBox"
   default_view: StyleableBoxView
 
-  defaults: ->
-    return _.extend {}, super(), {
-      children: []
-      css_properties: {}
-      orientation: 'vertical'
-    }
-
-  children: () ->
-    return @get('children')
+  @define {
+    css_properties: [p.Dict, null]
+    orientation:    [p.String, 'vertical']
```

Comment:
You could use `p.Orientation` enum property. Unless this orientation has more choices.